inlong-manager/manager-test/src/main/resources/h2/apache_inlong_manager.sql

```diff
@@ -329,6 +330,7 @@ CREATE TABLE IF NOT EXISTS `stream_source`
     `uuid`                varchar(30)           DEFAULT NULL COMMENT 'Mac uuid of the agent running the task',
     `data_node_name`      varchar(128)          DEFAULT NULL COMMENT 'Node name, which links to data_node table',
     `inlong_cluster_name` varchar(128)          DEFAULT NULL COMMENT 'Cluster name of the agent running the task',
+    `inlong_cluster_node_tag` varchar(128)      DEFAULT NULL COMMENT 'Cluster node tag',
     `serialization_type`  varchar(20)           DEFAULT NULL COMMENT 'Serialization type, support: csv, json, canal, avro, etc',
```

Comment:
`node_tags` and `inlong_cluster_node_tag` are two fields causing confusion the tag of the cluster.

Reply:
inlong_cluster_node_tag represent this source task must match those node who has these tag.
node_tag represent this node has this tag.

Reply:
> inlong_cluster_node_tag represent this source task must match those node who has these tag.
> node_tag represent this node has this tag.

What to do if different cluster name have the same tag?

Reply:
this node tag is only to filter task to delivery to which agent node.

Reply:
> this node tag is only to filter task to delivery to which agent node.

GET